static/js/components/Dashboard.js

```diff
@@ -14,7 +14,7 @@ class Dashboard extends React.Component {
 
   render() {
     const { profile, dashboard, expander, dispatch } = this.props;
-    let imageUrl = (SETTINGS.edx_base_url + '/static/images/profiles/default_120.png').
+    let imageUrl = `${SETTINGS.edx_base_url}/static/images/profiles/default_120.png`.
```

Comment:
does `path` do anything for us? It's part of node, so it wouldn't run in the browser, but if there were some way to signify that an expression should be evaluated and then substituted when compiling / building the JS we could maybe use that? idk that's far-fetched haha